src/components/cms/Dashboard/MyAnalytics.js

```diff
@@ -71,7 +78,9 @@ class MyAnalytics extends Component {
           <div>
             {skillUsageCount !== 0 && (
               <React.Fragment>
-                <SubTitle>Skill Usage Distribution</SubTitle>
+                <SubTitle marginLeft={'1.4rem'}>
```

Comment:
If want to use ` marginLeft={'1.4rem'}` then use `style={{ marginLeft: 1.4rem }}>`

Reply:
@samagragupta It is a shared component.

Reply:
@samagragupta Corrected it. Made it `margin-left` instead of `margin`. It is a shared component and we are getting rid of inline styles. So it's better to keep it scalable for the future.